benchmarks/tpch/justfile

```
@@ -0,0 +1,48 @@
+set fallback
+VENV := ".venv"
+VENV_BIN := ".venv/bin"
+
+
+@venv:
+  @python3 -m pip install virtualenv > /dev/null
+  @python3 -m virtualenv {{VENV}} > /dev/null
+
+## Set up virtual environment and install requirements
+@requirements: venv
+  {{VENV_BIN}}/python -m pip install --upgrade pip > /dev/null
+  {{VENV_BIN}}/pip install -r requirements.txt > /dev/null
+  
+clean-tpch-dbgen:
+  make -C tpch-dbgen clean
+
+clean-venv:
+  rm -r {{VENV}}
+
+clean-tables:
+  rm -r tables_scale/
+
+clean: clean-tpch-dbgen clean-venv
+
+gen_scale_factor n: requirements
+  make -C tpch-dbgen all
+  cd tpch-dbgen && ./dbgen -vf -s {{n}} && cd ..
```

```suggestion
  pushd tpch-dbgen && ./dbgen -vf -s {{n}} && popd
```
https://just.systems/man/en/chapter_45.html

changing back isn't needed but makes it a bit more edit proof if you convert it to a bash script.